src/pages/register/index.js

```diff
@@ -8,12 +8,42 @@ import "./register.css";
 const Register = () => {
 	const { onRegister } = useAuth();
 	const [formData, setFormData] = useState({ email: "", password: "" });
+	const [isEmailValid, setIsEmailValid] = useState(null)
+	const [isPasswordValid, setIsPasswordValid] = useState(null)
+	const [isEmailTaken, setIsEmailTaken] = useState(null)
```

Comment:
Boolean state should be initialised as either true or false, I'd change these nulls to false

Reply:
Would have to set them to true by default then so the error messages don't show up on first load only after submit has complaints.

Reply:
Ah, good point. In that case you can ignore my comment on this